msal/src/Microsoft.Identity.Client/TokenCache.cs

```diff
@@ -945,26 +959,39 @@ internal void ClearCache()
         {
```

Comment:
two things:
1 - in adal, this method `ClearCache()` is public - should it be public here as well?
2 - rename to `Clear()` to match adal
3 - can this be moved to core? seems to be doing the same thing in both libraries, so should be in core

Reply:
1. it is not public in msal. 3. no TokeCache is implementations are different in adal an msal